src/pthreads.h

```diff
@@ -138,6 +138,8 @@ ZEND_END_MODULE_GLOBALS(pthreads)
 #define PTHREADS_PG(ls, v) PTHREADS_FETCH_CTX(ls, core_globals_id, php_core_globals*, v)
 #define PTHREADS_EG_ALL(ls) PTHREADS_FETCH_ALL(ls, executor_globals_id, zend_executor_globals*)
 
+#define PTHREADS_THROW_MSG(errno, msg) zend_throw_exception_ex(spl_ce_RuntimeException, (errno), (msg))
```

Comment:
Not much is really gained by making this a macro, particularly in one of the main pthreads headers (since it is only used in the socket.c file).

Reply:
Thought about later extensibility, but the more I think about it, I'll change that.